docs/how-to/h-deploy-azure.md

```diff
@@ -0,0 +1,239 @@
+# How to deploy on Azure
+
+[Azure](https://azure.com/) is the cloud computing platform developed by Microsoft. It has management, access and development of applications and services to individuals, companies, and governments through its global infrastructure. Access the Azure web console at [portal.azure.com](https://portal.azure.com/).
+
+## Summary
+* [Install Azure and Juju tooling](#install-client-environment)
+  * [Authenticate](#authenticate)
+* [Bootstrap Juju controller on Azure](#bootstrap-juju-controller-on-azure)
+* [Deploy charms](#deploy-charms)
+* [Clean up](#clean-up)
+
+---
+
+## Install Client Environment
+
+> **WARNING**: The current limitations:
+> * Only supported starting Juju 3.6 (currently edge)
+> * Juju cli should be on Azure VM for it to be able to reach cloud metadata endpoint.
+> * Managed Identity and the Juju resources should be on the same Azure subscription
+> * The current setup has been tested on Ubuntu 22.04+
+
+### Juju 
+
+Install Juju via snap:
+
+```shell
+sudo snap install juju --channel 3.6/edge
+```
+
+Check that the Juju version is correctly installed:
+
+```shell
+juju version
+```
+
+which should show:
+
+```shell
+3.6-rc1-genericlinux-amd64
+```
```

Comment:
In this specific case, it is actually very important since the process here would not work on other versions, even on the `3.6` track at the time of writing